openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py

```diff
@@ -74,12 +74,43 @@ def nrcan15_site_term(ctx, imt, me, si, ta, ph, kind):
     me[:] = np.log(exp_mean * fa)
 
 
+def ceus2020_site_term(
+        ctx, imt, me, si, ta, phi, wimp, ref_vs30, ref_pga, usgs=False):
+    """
+    This function adds the Stewart et al. (2020; EQS) site term that uses as
+    a reference 760 m/s.
+
+    :param wimp:
+        The 'wimp' factor in eq. 5 of Stewart et al. (2020)
+    :param ref_vs30:
+        The reference Vs30 value
+    :param ref_pga:
+        The reference PGA value computed for a vs30 corresponding to `ref_vs30`
+    """
+
+    from openquake.hazardlib.gsim.mgmpe.stewart2020 import (
+        stewart2020_linear_scaling)
+    from openquake.hazardlib.gsim.mgmpe.hashash2020 import (
+        hashash2020_non_linear_scaling)
+
```

Comment:
Why the imports inside the function?

Reply:
These imports are required only when this function is used, so I thought it could be more convenient to keep them inside. I reverted their position to the usual one.